packages/eslint-plugin/test/import-relative-within-subpackage.spec.js

```diff
@@ -57,25 +57,5 @@ ruleTester.run('import-relative-within-subpackage', rule, {
         'Do not use an alias to import from core/subpackage from code inside core/subpackage. Instead, use a relative import',
       ],
     },
-    {
```

Comment:
These are the tests for this special case, but there should be changes to the eslint rule itself, I think.

Reply:
DUH sorry. The actual kubernetes-specific package parsing logic is removed now.